apps/scan/backend/src/electrical_testing/server.ts

```diff
@@ -4,14 +4,70 @@ import { PORT } from '../globals';
 import { buildApp } from './app';
 import { cardReadAndUsbDriveWriteLoop, printAndScanLoop } from './background';
 import { ServerContext } from './context';
+import {
+  createSimpleScannerClient,
+  SimpleScannerClient,
+} from './simple_scanner_client';
+
+export interface ElectricalTestingServerContext extends ServerContext {
+  scannerClient: SimpleScannerClient;
+  controller: AbortController;
+}
```

Comment:
nit: Maybe colocate this in the `electrical_testing/context.ts` file

Reply:
[4628165](https://github.com/votingworks/vxsuite/pull/6048/commits/46281654e610e4444d692d4d8b69d7bbc94248c5)